doc/modules/core.rst

```diff
@@ -173,25 +173,14 @@ computations.
 
 The :py:class:`~spikeinterface.core.SortingAnalyzer` provides a convenient API to access the underlying
 :py:class:`~spikeinterface.core.BaseSorting` and :py:class:`~spikeinterface.core.BaseRecording` information,
-and it supports several extensions (derived from the :py:class:`~spikeinterface.core.AnalyzerExtension` class)
-to perform further analysis.
+and it supports several **extensions** (derived from the :py:class:`~spikeinterface.core.AnalyzerExtension` class)
+to perform further analysis, such as calculating :code:`waveforms` and :code:`templates`.
 
-Importantly, the :py:class:`~spikeinterface.core.SortingAnalyzer` handles the *sparsity* (see [Sparsity]_ section), i.e.,
+Importantly, the :py:class:`~spikeinterface.core.SortingAnalyzer` handles the *sparsity* (see :ref:`Sparsity` section), i.e.,
```

Comment:
Can we add here that the `return_scaled` is also handled at the analyzer level, after #2684 ?